content/en/tracing/trace_collection/automatic_instrumentation/dd_libraries/cpp.md

```diff
@@ -23,153 +23,169 @@ further_reading:
   tag: "Advanced Usage"
   text: "Advanced Usage"
 ---
-**Note**: C++ does not provide integrations for OOTB instrumentation, but it's used by Proxy tracing such as [Envoy][1] and [Nginx][2]. For compatibility requirements for the C++ Tracer, visit the [Compatibility Requirements][3] page.
+
+<div class="alert alert-warning">
+  <strong>Note:</strong> C++ does not provide integrations for automatic instrumentation, but it's used by Proxy tracing such as <a href="/tracing/setup/envoy/">Envoy</a> and <a href="/tracing/setup/nginx/">Nginx</a>.
+</div>
+
+## Compatibility requirements
+The C++ tracing library requires C++17 toolchain to build. For a full list of Datadog's tracing library compatiblity requirements and processor architecture support, visit the [Compatibility Requirements][3] page.
```

Comment:
```suggestion
The C++ tracing library requires a C++17 compiler to build. For a full list of Datadog's tracing library compatibility requirements and processor architecture support, visit the [Compatibility Requirements][3] page.
```
^ Is it more accurate to say this needs a C++17 compiler instead of toolchain?

Reply:
A toolchain is a set of tools (such as a compiler, linker, and assembler). A compiler transform code to binary format. A compiler on its own will no generate the static/shared library, I do think toolchain is more appropriate.